cli/src/base/AdvancedTransactionsCommandBase.ts

```diff
@@ -266,10 +275,19 @@ export default abstract class AdvancedTransactionsCommandBase extends AccountsCo
     return unsigned
   }
 
-  async getWeight(call: Call): Promise<number> {
+  async getWeight(call: Call): Promise<{
+    refTime: string | number
+    proofSize: string | number
+  }> {
     const callData = this.getOriginalApi().tx(call)
     const paymentWeight = await this.getOriginalApi().rpc.payment.queryInfo(callData.toHex())
-    return paymentWeight.weight.toNumber()
+    const proofSize = 0
+    const value = {
+      refTime: paymentWeight.weight.toNumber(),
```

Comment:
I think this is still not correct. My guess it would probably also throw an exception decoding the queryInfo() result? But not certain because in other places we do [tx.paymentInfo()](https://github.com/Joystream/joystream/blob/332e22d0b06a27c76c4e7a48db8662072a82dfdb/cli/src/Api.ts#L146) which also return a `DispatchInfo` type but we use the `partialFee` field not the weight.

The problem is despite updating version of tx-wrapper we are still at the workspace level forcing resolution for polkadot-js api v8.9.1 and so the returned type from `queryInfo()` is expected to be:

```ts
/** @name RuntimeDispatchInfo */
export interface RuntimeDispatchInfo extends Struct {
    readonly weight: Weight;
    readonly class: DispatchClass;
    readonly partialFee: Balance;
}

/** @name Weight */
export interface Weight extends u64 {}
```
But infact the weight type is not a u64.

We can see the warning when yarn is installing dependencies:

```sh
warning Resolution field "@polkadot/api@8.9.1" is incompatible with requested version "@polkadot/api@^10.9.1"
```

When I originally refactored some code in repo that relates to estimating fees I didn't give it the attention it needed. For example in this fee-profile I now see my error: https://github.com/Joystream/joystream/blob/332e22d0b06a27c76c4e7a48db8662072a82dfdb/cli/src/base/FeeProfileCommandBase.ts#L102